deploy/docker-compose.yml

```diff
@@ -0,0 +1,346 @@
+version: "3"
+
+services:
+  zookeeper:
+    image: zookeeper:3.6
+    restart: unless-stopped
+    hostname: zookeeper
+    container_name: zookeeper
+    ports:
+      - "12181:2181"
+    environment:
+      ZOO_MY_ID: 1
+      ZOO_STANDALONE_ENABLED: 'true'
+    expose:
+      - 2181
+    networks:
+      - kafka
+    volumes:
+      - type: volume
+        source: zookeeper_data
+        target: /data
+        volume:
+          nocopy: true
+      - type: volume
+        source: zookeeper_datalog
+        target: /datalog
+        volume:
+          nocopy: true
+    healthcheck:
+      test: [
+          "CMD",
+          "nc",
+          "-z",
+          "127.0.0.1",
+          "2181"
+      ]
+      interval: 10s
+      timeout: 5s
+      retries: 30
+
+  cruisecontrolmetrics:
+    image: ghcr.io/banzaicloud/cruise-control:2.5.79
```

Comment:
Should we bump this to 2.5.80?

Reply:
We could, although there is not AP change between the two versions. I'll bump the version but in a separate PR.

Reply:
Fixed.